rebar.config

```diff
@@ -1,8 +1,7 @@
-{require_otp_vsn, "R15|R16|(17.*)|(18.*)"}.
```

Comment:
Yes! Just make it forwards compatible. It is more likely to work than not, given that sv uses fairly few "magic" features.